apps/coscrad-frontend/src/components/image-in-content/image-in-content.tsx

```diff
@@ -0,0 +1,45 @@
+export type ImageAsset = {
```

Comment:
Throwing out ideas here, but this may be too weird or too much complexity for now

Reply:
We're leaving this in as a placeholder for the future of digital asset management